packages/wonder-blocks-switch/src/themes/default.ts

```diff
@@ -1,29 +1,35 @@
 import * as tokens from "@khanacademy/wonder-blocks-tokens";
 
+const {semanticColor} = tokens;
+
+// The color of the switch is based on the action color.
+const action = semanticColor.action.outlined.progressive;
+
 const theme = {
     color: {
         bg: {
             switch: {
-                off: tokens.color.offBlack50,
-                disabledOff: tokens.color.offBlack32,
-                activeOff: tokens.color.offBlack64,
-                on: tokens.color.blue,
-                disabledOn: tokens.color.fadedBlue,
-                activeOn: tokens.color.activeBlue,
+                off: action.default.border,
+                disabledOff: semanticColor.action.disabled.default,
+                activeOff: semanticColor.text.secondary,
```

Comment:
Would it make sense to use the primitive token directly here or a different semantic token? If we were to change the secondary text color value, would we expect things that are not text to change as well? 

Same question about using the border color for `theme.color.bg.switch.off`!

Reply:
That's an excellent question! I think using primitive token would make sense here for now.

Side note: I wonder if we should change `icon.primary` to use the new `offBlack72` token. wdyt? should we raise this to Caitlyn?

Reply:
I was wondering something similar! At one point, I came across text with an icon in `webapp`. The text now uses `text.secondary` (offBlack72), so I thought I'd use `icon.secondary` for the icon to match it. For icons, `secondary` is darker than `primary`! Do we want to swap them so they are consistent with the text tokens (ie. use primary text with primary icon color, use secondary test with secondary icon color)? Also, I wonder what the use cases are for using the secondary icon color (currently `offBlack`), maybe we don't need both?

I think this is something we can bring up with Caitlyn for the token work!